### DISCUSSION TITLE
CropScaleMode.Crop on large image returns stretched image?

### DISCUSSION THREAD

**Discussion by RickDeBay** [saucecontrol/PhotoSauce#75] [Q&A]
I cropped a 5168x3448 image to 200x200, and the result was the original image shrunk to 200x200 instead of a very small piece of the image.
ProcessImageSettings were width 200, height 200, and ResizeMode Crop.
Is this to be expected or is this a bug?

**Comment by saucecontrol** [comment]
That is the expected behavior.  The default [`Crop`](https://docs.photosauce.net/api/PhotoSauce.MagicScaler.ProcessImageSettings.html#PhotoSauce_MagicScaler_ProcessImageSettings_Crop) rectangle is `0,0,0,0` which means to process the entire image.  With a `Width` and/or `Height` smaller than the original, the result will be a scaled image.  `ResizeMode` controls whether the image will be distorted to fit the bounds given or cropped to preserve the original pixel aspect ratio, and since your given bounds did not match the ratio of the input image, the left and right sides would have been cropped.

If you want to crop without resizing, you should leave `Width` and `Height` at their default values and supply a `Crop` rectangle that indicates your area of interest.  For example `new ProcessImageSettings { Crop = new Rectangle(100,100,200,200) }` will crop a 200x200 square from the input image starting at an offset of x=100,y=100.
